cypress/e2e/fact_spec.cy.js

```diff
@@ -0,0 +1,27 @@
+describe('visiting the fact page', () => {
+
+  it('should show a random fact upon visit', () => {
+    cy.intercept('https://catfact.ninja/fact', {
+      status: 200,
+      fixture: 'cat-facts'
+    })
+    cy.visit('http://localhost:3000/')
+    cy.get('button[class=fact-btn]').click()
+    cy.get('p').contains('Cats can see like wicked good')
+  })
+
+  it('should have two buttons', () => {
+    cy.visit('http://localhost:3000/fact')
+    cy.get('button[class=fav-card-btn]')
+    cy.get('button[class=new-fact-btn]')
+  })
```

Comment:
You can add ".should('be.visible') here after "getting" the buttons

Reply:
Thanks. I was curious to if there were something more than just a "get" to do there